### PR TITLE
Use Result::Sat instead of BenchmarkStatus in printers.

### PR DESCRIPTION
**Pull request by abdoo8080**
This PR modifies the printers to use `Result::Sat`, "internal" version of `BenchmarkStatus`, for printing benchmark status commands.

### CODE REVIEW

File: src/smt/command.cpp [hunk 1]
```diff
@@ -2977,7 +2977,17 @@ void SetBenchmarkStatusCommand::toStream(std::ostream& out,
                                          size_t dag,
                                          OutputLanguage language) const
 {
-  Printer::getPrinter(language)->toStreamCmdSetBenchmarkStatus(out, d_status);
+  Result::Sat status;
+  switch (d_status)
+  {
+    case BenchmarkStatus::SMT_SATISFIABLE: status = Result::SAT; break;
+    case BenchmarkStatus::SMT_UNSATISFIABLE:
+      status = Result::SAT_UNKNOWN;
```
Comment (ajreynol) [inline src/smt/command.cpp:2985]: I think this was swapped?

**Review by ajreynol — CHANGES_REQUESTED** [review]
One typo I think.

File: src/smt/command.cpp (hunk 1 above)
Comment (abdoo8080) [reply src/smt/command.cpp]: Fixed!

**Review by ajreynol — APPROVED** [review]
LGTM

**Pull request merged**